src/content/docs/auto-telemetry-pixie/pixie-data-security-overview.mdx

```diff
@@ -9,18 +9,25 @@ tags:
 metaDescription: 
 ---
 
-New Relic hosts Pixie, our open source observability tool for Kubernetes applications. Pixie fetches data flowing through a systems kernel via [eBPF](https://ebpf.io/) probes from a privileged state. If an application contains sensitive data, this data flow can potentially collect that data in plain text and save it to [NRDB](/docs/telemetry-data-platform/get-started/nrdb-horsepower-under-hood/). For example, Pixie gives insight into any data on your system, including raw HTTP requests in the kernel, allowing you to view all headers, query parameters, and HTTP bodies of a request.
+Auto-telemetry with Pixie is New Relic One's integration of Community Cloud for Pixie, a managed version of Pixie open source software. Auto-telemetry with Pixie therefore benefits from Pixie's approach to keeping data secure. The data that Pixie collects is stored entirely within your Kubernetes cluster. This data is not persisted outside of your environment, and will never be stored by Community Cloud for Pixie. This means that your sensitive data remains within your environment and control.
```

Comment:
"therefore" might be unnecessary

Reply:
"is not persisted" --> "does not persist"?

Reply:
I stuck with "therefore" because I'm trying to draw the connection from Pixie to Community Cloud with Pixie to Auto-telemetry with Pixie. Otherwise, accepted (with gratitude!) your other changes. Thanks!